demos/middleware/middleware.ts

```diff
@@ -24,7 +24,7 @@ export async function middleware(req: NextRequest) {
   if (pathname.startsWith('/static')) {
     // Unlike NextResponse.next(), this actually sends the request to the origin
     const res = await request.next()
-    const message = `This was static but has been transformed in ${req.geo?.city}`
+    const message = `This was static & old but has been transformed in ${req.geo?.city}`
```

Comment:
These tests are great. I wonder if it makes sense to test that `&amp;` would render as the text `&amp;`? The only use case for that that I can think of is if a code example was in a prop being set. Maybe not worth it. 🤔

Reply:
Done!